raiden-ts/src/types.ts

```diff
@@ -49,3 +49,15 @@ export interface RaidenEpicDeps {
   userDepositContract: UserDeposit;
   main?: { signer: Signer; address: Address };
 }
+
+export interface ChangeEvent<T, P> {
```

Comment:
Then you can just add the requirement, which makes it also compatible with the string literals from enums
```suggestion
export interface ChangeEvent<T extends string, P> {
```